Alarm support multiple scope (#4769)
apache/skywalking: docs/en/setup/backend/backend-alarm.md

### List of all potential metrics name
The metrics names are defined in official [OAL scripts](../../guides/backend-oal-scripts.md), right now 
metrics from **Service**, **Service Instance**, **Endpoint**, **Service Relation**, **Service Instance Relation**, **Endpoint Relation** scopes could be used in Alarm, and the **Database access** same with **Service** scope.

Submit issue or pull request if you want to support any other scope in alarm.

## Webhook
Webhook requires the peer is a web container. The alarm message will send through HTTP post by `application/json` content type. The JSON format is based on `List<org.apache.skywalking.oap.server.core.alarm.AlarmMessage>` with following key information.
- **scopeId**, **scope**. All scopes are defined in org.apache.skywalking.oap.server.core.source.DefaultScopeDefine.
- **name**. Target scope entity name. Please follow [Entity name define](#entity-name).

> - **id0**. The ID of scope entity, matched the name. When using relation scope, it will be the source entity ID.

Reviewer: ```suggestion
- **id0**. The ID of the scope entity matched the name. When using relation scope, it is the source entity ID.
```

> - **id1**. When using relation scope, it will be the dest entity ID. Otherwise it will be empty.

Reviewer: ```suggestion
- **id1**. When using relation scope, it will be the dest entity ID. Otherwise, it is empty.
```
